### ISSUE TITLE
Codestyling

### ISSUE THREAD

**Issue by Niko9911** [justbetter/magento2-sentry#30]
To enforce codestyling, project should utilize phpcsfixer. I will be implementing this into develop branch after #26 has been merged.

**Comment by rakibabu** [comment]
@Niko9911 ,

The PR has been merged into develop.

**Comment by Niko9911** [comment]
Yes, I see, but issue about codestyle standards is still valid @rakibabu
